src/core/Akka/Serialization/Serializer.cs

```diff
@@ -121,7 +121,17 @@ public static int GetSerializerIdentifierFromConfig(Type type, ExtendedActorSyst
         {
```

Comment:
No. `Serializer.Identifier` is still valid. What only have changed, is that beside of having statically assigned number, now it can be a configurable value instead.